docs/architecture/rook-ceph-install.md

```
@@ -0,0 +1,432 @@
+
+### Step 3: Verify Operator Deployment
+
+Confirm the Rook operator is running:
+
+```bash
+kubectl -n rook-ceph get pods
+```
+
+**Expected output:**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)